.eslintignore

```diff
@@ -1,5 +1,6 @@
 node_modules
 examples/social-network/frontend
+examples/ws-server
```

Comment:
If you end up needing the `ws-server` example, I recommend you move this line to a `.eslintignore` file inside `examples/ws-server`. Probably we should do the same with the line above.

Reply:
I am removing the `ws-server` example as I don't need it anymore.